tests/test_check_function.py

```diff
@@ -82,16 +82,17 @@ def test_args_kwargs_check_function_failing_not_specified():
     with pytest.raises(TF, match=msg):
         x.check_args(['kwargs', 'c'])
 
+@pytest.mark.debug
 def test_args_kwargs_check_function_failing_not_correct():
     s = setup_state(pec = 'def my_fun(a, b, *args, **kwargs): pass',
                     sol_code = 'my_fun(1, 2, 3, 4, c = 5)',
                     stu_code = 'my_fun(1, 2, 4, 5, c = 6)')
     x = s.check_function('my_fun')
-    with pytest.raises(TF):
+    with pytest.raises(TF, match='first argument matched to the star'):
         x.check_args(['args', 0]).has_equal_value()
-    with pytest.raises(TF):
+    with pytest.raises(TF, match='second argument matched to the star'):
         x.check_args(['args', 1]).has_equal_value()
-    with pytest.raises(TF):
+    with pytest.raises(TF, match='Check the argument `c` of the'):
```

Comment:
Current: 
Check the argument `c` matched through the kwargs of the first function call of ...

Will the update message render as: 
Check the argument `c` of the kwargs of the first function call of

Reply:
No it will update to:

"Check the argument `c` of the first function call ..."

Reply:
I believe it would be more appropriate to say: 

> Check the argument `c` passed as a variable length keyword argument to the first function call of ...

Reply:
@sumedh10 I believe it doesn't, as students don't really care. a keyword argument always has to be passed with its name, so why go through the trouble of mentioning that? The case of variable length positional arguments is different, because there it is possible that other arguments have been specified by position ealier in the list of arguments.

Reply:
I mentioned it because `c` is technically not an argument to the function. 

>  I believe it doesn't, as students don't really care.
But I do agree with this statement.